catalog/spatial/geocoding/spatial-geocoding-endpoint/pom.xml

```diff
@@ -45,6 +50,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            commons-text
+                        </Embed-Dependency>
```

Comment:
❓ Did this need to be embedded? I feel like we should be exposing an apache utility bundle by now, but I've got no idea 🤷‍♂

Reply:
Not `commons-text`. Not yet, anyway.